pandas/core/arrays/base.py

```diff
@@ -354,6 +355,19 @@ def __iter__(self):
         for i in range(len(self)):
             yield self[i]
 
+    def __contains__(self, item) -> bool:
+        """
+        Return for `item in self`.
+        """
+        # comparisons of any item to pd.NA always return pd.NA, so e.g. "a" in [pd.NA]
```

Comment:
can you add this issue number here for a future refernce

Reply:
Ok